health-services/project-factory/src/server/utils/transforms/projectTypeUtils.ts

```
@@ -93,35 +93,33 @@
 Enrich project details from campaign details.
 */
 export const enrichProjectDetailsFromCampaignDetails = (
-  CampaignDetails: any = {},
-  projectTypeObject: any = {}
+  CampaignDetails: any = {}
 ) => {
-  var { tenantId, projectType, startDate, endDate, campaignName } =
-    CampaignDetails;
+  var { tenantId, projectType, startDate, endDate, campaignName, campaignNumber } = CampaignDetails;
   logger.info("campaign transformation for project type : " + projectType);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)